src/controllers/pages/darknodeStatsPage/DarknodeStatsPage.tsx

```diff
@@ -58,6 +65,56 @@ export const DarknodeStatsPage = () => {
     const { renNetwork } = Web3Container.useContainer();
     const { darknodes, fetchDarknodes } = MapContainer.useContainer();
 
+    useEffect(() => {
+        const sse = new EventSource(
+            "https://api.zapper.fi/v1/balances?addresses%5B%5D=0x7556aea47efc2a628e7eebc325de44572454b1e9&addresses%5B%5D=0x5291fbb0ee9f51225f0928ff6a83108c86327636&api_key=96e0cc51-a62e-42ca-acee-910ea7d2a241",
```

Comment:
Should this api key be private? If so it can be stored as an environment variable. It might also make sense to store the addresses as variables and construct the URL dynamically, to make changing the addresses easier in the future.

Reply:
I thought the same but apparently Zapper provides a public API key to use

Reply:
Addresses as variables is a good point though

Reply:
The API key is public and pushed a commit to store the addresses as variable

Reply:
> 

done